### ISSUE TITLE
Missing License Information

### ISSUE THREAD

**Issue by Zarepheth** [Phylogeny/JEIEnchantmentInfo#3]
**Versions**
* Minecraft: 1.16.3
* Forge: 1.16.3-34.1.42
* JEI: 1.16.3-7.6.0.51
* JEI Enchantment Info: 1.16.1-1.2.0

As the title says:
```
Failure message: Missing License Information in file Mod File: C:\Users\...\Documents\Curse\Minecraft\Instances\Fiefdom\mods\JEIEnchantmentInfo-1.16.1-1.2.0.jar
```

From some similar threads for other mods, it sounds like Forge for Minecraft 1.16.2 and beyond require mod authors to add license information to their mod `.toml` files. Failure to find license info results in a crash. :(

**Comment by GnomTocha** [comment]
![image](https://user-images.githubusercontent.com/75666317/102698507-ec0ba300-424e-11eb-9c83-fe110e40d5c0.png)
this is 2 mod where i see similar)

**Comment by Plasticable** [comment]
I have the same problem. Does anyone have a solution?

**Comment by Phylogeny** [comment]
This is fixed as of 1.16.4-1.2.1
